includes/extra_configures/enable_error_logging.php

```diff
@@ -96,7 +96,7 @@ function zen_fatal_error_handler()
     $last_error = error_get_last();
 
     if (!empty($last_error) && in_array($last_error['type'], [E_ERROR, E_USER_ERROR, E_PARSE])) {
-        $message = date('[d-M-Y H:i:s e]') . ' Request URI: ' . $_SERVER['REQUEST_URI'] . ', IP address: ' . (isset($_SERVER['REMOTE_ADDR']) ? $_SERVER['REMOTE_ADDR'] : 'not set') . PHP_EOL;
+        $message = date('[d-M-Y H:i:s e]') . ' Request URI: ' . (isset($_SERVER['REQUEST_URI']) ? $_SERVER['REQUEST_URI'] : 'not set') . ', IP address: ' . (isset($_SERVER['REMOTE_ADDR']) ? $_SERVER['REMOTE_ADDR'] : 'not set') . PHP_EOL;
```

Comment:
couldn't both of these checks be done via a coalesce operator? `??`

Reply:
Agreed.